tests/std/tests/P0896R4_ranges_iterator_machinery/test.cpp

```diff
@@ -3474,9 +3474,19 @@ namespace vso1121031 {
     STATIC_ASSERT(!has_member_value_type<indirectly_readable_traits<iterish<float const volatile>>>);
 } // namespace vso1121031
 
+constexpr bool test_lwg_XXX() {

```

Comment:
I'd like to annotate (here and in the header) with the proper issue number before merging.